src/backend/distributed/citus.sql

```diff
@@ -103,6 +103,27 @@ SET search_path = 'pg_catalog';
 
 /* master_* functions */
 
+CREATE FUNCTION citus_table_size(relation_name text)
```

Comment:
To supplement this, and the above comment on adding regression tests, [here's an example commit](https://github.com/citusdata/citus/commit/7cff8719c2166ba43aca36559b377f643f4f6665). It should show you all the places where the version number needs to be updated, and an example of how to add tests.